library_app.rb

```diff
@@ -0,0 +1,34 @@
+require 'lib/library'
+
+library = Library.new
+
+john = Author.new(name: 'John')
```

Comment:
check ffaker gem and move creating new classes to seed.rb or into a separate service